src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,9 @@
+import classnames from 'classnames';
+
+import * as React from 'react';
+import style from './Divider.scss';
+
+export function Divider(props: any) {
```

Comment:
Should define className as an optional prop, e.g:
``` typescript
export interface IProps {
  className?: string;
}
```

This way you can know the the Component can use it, without looking at the code for the Component.